spec/features/pom_caseload_feature_spec.rb

```diff
@@ -167,6 +167,12 @@
       end
     end
 
+    it 'can be sorted by responsibility', :js do
+      sleep 10
+      click_link 'Role'
+      sleep 50
+    end
```

Comment:
These long sleeps are going to have a real impact on the length of our test runs. This alone will add a whole minute ☹️

Do these sleeps really need to be this long?